eks/cluster-loader/clusterloader2/cluster-loader.gotmpl

```diff
@@ -44,6 +44,7 @@ metadata:
   name: clusterloader2
   namespace: clusterloader2
 spec:
+  backoffLimit: 0
```

Comment:
What does this do?

Reply:
If the test doesn't pass for whatever reason, without this, it will attempt to restart the init container. Restarting the init container will do another CL2 test, which we don't want.

Reply:
Nice!